pkg/formatter/table.go

```diff
@@ -33,35 +34,50 @@ const (
 	EmptyValuePlaceholder = "-"
 )
 
+var (
+	NodeReportHeader = []string{"Node Name", "Chef Version", "Operating System", "Cookbooks"}
+)
+
 func WriteNodeReport(records []*reporting.NodeReportItem) {
 	termWidth, _, err := terminal.GetSize(int(os.Stdout.Fd()))
 	if err != nil {
 		termWidth = MinTermWidth
 	}
+	buffer := bytes.NewBufferString("")
 
-	table := tablewriter.NewWriter(os.Stdout)
+	// Let's look at content to pre-determine the best column widths
+	table := tablewriter.NewWriter(buffer)
 	table.SetAutoWrapText(true)
 	table.SetReflowDuringAutoWrap(true)
-	table.SetColMinWidth(0, int(float64(termWidth)*0.30)) // fqdns can get pretty long
-	table.SetColMinWidth(1, int(float64(termWidth)*0.10)) // chef version is tiny
-	table.SetColMinWidth(2, int(float64(termWidth)*0.15)) // OS+version string
-	// Help prevent any one column from expanding to fill all available space:
-	table.SetColWidth(int(float64(termWidth) * 0.35))
-	table.SetHeader([]string{"Node Name", "Chef Version", "Operating System", "Cookbooks"})
-	table.SetAutoFormatHeaders(false) // Don't make our headers capitalize
+	table.SetHeader(NodeReportHeader)
+	table.SetAutoFormatHeaders(false) // Don't make our headers capitalized
```

Comment:
What is this for? The headers are actually capitalized, aren't they? 🤔

Reply:
Without this it makes them ALL CAPS